server/routes/refresh.ts

```diff
@@ -21,5 +21,5 @@ export default eventHandler(async (event) => {
   await deleteByUserId();
   await save();
 
-  return user.toJSON();
+  return omit(user.toObject(), ["privateKey"]);
```

Comment:
Use flattenMaps, because meta will not be transformed to object
```suggestion
  return omit(user.toObject({ flattenMaps: true }), ["privateKey"]);
```